src/Forms/ClientForm.php

```diff
@@ -37,12 +36,15 @@ class ClientForm extends Form
     final public const REGEX_URI = '/^[^:]+:\/\/?[^\s\/$.?#].[^\s]*$/';
 
     /**
-     * Must have http:// or https:// scheme, and at least one 'domain.top-level-domain' pair, or more subdomains.
-     * Top-level-domain may end with '.'.
-     * No reserved chars allowed, meaning no userinfo, path, query or fragment components. May end with port number.
+     * Must have:
+     * - http:// or https:// scheme
+     * - at least one 'domain, or more subdomains, or a domain with a wild card for subdomains
```

Comment:
I don't think a wildcard can ever end up in Origin header. What is the meaning of "...or a domain with a wild card for subdomains"?

Reply:
Removed obsolete comment change